src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/BundleSlingInitialContentJarEntryExtractor.java

```diff
@@ -70,7 +67,8 @@ class BundleSlingInitialContentJarEntryExtractor {
      */
     void extractAndAddToAssembler(@NotNull BundleSlingInitialContentExtractContext context,
                                   @NotNull SlingInitialContentBundleEntryMetaData slingInitialContentBundleEntryMetaData,
-                                  @NotNull Set<SlingInitialContentBundleEntryMetaData> collectedSlingInitialContentBundleEntries) throws IOException, ConverterException {
+                                  @NotNull Set<SlingInitialContentBundleEntryMetaData> collectedSlingInitialContentBundleEntries,
+                                  @NotNull boolean isDefaultContentXmlFile) throws IOException, ConverterException {
```

Comment:
Does `@NotNull` make sense for a primitive value?

Reply:
nope it doesn't :)
sonar will complain about it.....